Exiled.API/Features/Badge.cs

```diff
@@ -52,5 +52,14 @@ public Badge(string text, string color, int type, bool isGlobal = false)
         /// </summary>
         /// <returns>A string containing Badge-related data.</returns>
         public override string ToString() => $"{Text} ({Color}) [{Type}] *{IsGlobal}*";
+
+        /// <summary>
+        /// Clones the current <see cref="Badge"/> object.
+        /// </summary>
+        /// <returns>A new <see cref="Badge"/> object.</returns>
+        public Badge Clone()
+        {
+            return new(Text, Color, Type, IsGlobal);
```

Comment:
Why dont you use lambda?

Reply:
That is a good question! I may have forgot it existed. will fix later